### ISSUE TITLE
PCBway for the touch panel

### ISSUE THREAD

**Issue by KeLaiFu** [plinkysynth/plinky-expander#2]
In some of the documentation it mentions using PCBway for the touch panel, and jlc for the rest, is there a reason for this? I would have assumed they both produce boards with the same functionality, however, PCBway is a lot more expensive so I thought I'd check before I get any boards made!

**Comment by mmalex** [comment]
sorry, that is confusing! I've used both, and jlc is in general cheaper. they both work :) I think the main issue is that JLC won't let you specify the LEDs for the front panel (of plinky or the expander!) because they don't stock those LEDs. whereas I was able to get pcbway to buy the relevant LEDs and fit them at the factory.

if you don't mind hand soldering the small number of LEDs on the expander, JLC is the way to go.

for the main plinky front panel, solding all 72 leds can be quite time consuming, so the main production run of those front panels was done at pcbway.

**Comment by KeLaiFu** [comment]
Brilliant, thanks for the reply. Is there any difference in the finish on the PCBs from the 2 factories? I had black PCB panels from jlc in the past and they were a matt texture, which I don't think would suit something that's in contact with oily hands for a living. But they may have changed their finish as it was years ago now.

**Comment by mmalex** [comment]
yeah, the pcbway lacquer seems a bit thicker, and maybe shinier. however I've found that the finish, printing, and color from JLC is really inconsistent - never really bad, per se, but no two runs are even remotely similar. :( so... theres an element of luck I think. (Ive only done one run from pcbway so can't compare their variance)

**Comment by KeLaiFu** [comment]
that lines up with what I have seen too. For the record, all the colors apart from black are glossy from jlc, so I may build a nice red plinky.
